### ISSUE TITLE
Evaluation metric

### ISSUE THREAD

**Issue by KhanhNgoDuy** [siyuhuang/QuantArt#13]
Hi, can you share the code for evaluating the model on metrics like FID and LPIPS?
Currently I'm using the FID and LPIPS results from [ArtFID](https://github.com/matthias-wright/art-fid), but not sure if this matches yours.

**Comment by shehuiyaoa11y** [comment]
Hello, the pre-training weights landscape2art provided by the author are no longer valid, can you help me? Provide that weight. Thank you!

**Comment by siyuhuang** [comment]
> Hello, the pre-training weights landscape2art provided by the author are no longer valid, can you help me? Provide that weight. Thank you!

I have fixed links to the pretrained models, but I cannot guarantee their weights are 100% correct. Feel free to report any issues!
